timetomodel/speccing.py

```diff
@@ -94,30 +116,34 @@ def _load_series(self) -> pd.Series:
     def load_series(
         self,
         expected_frequency: timedelta,
+        time_window: Tuple[datetime, datetime] = None,
         transform_features: bool = False,
-        check_datetime_index_window: Optional[Tuple[datetime, datetime]] = None,
+        check_time_window: bool = False,
     ) -> pd.Series:
         """Load the series data, check compatibility of series data with model specs
-           and perform feature transformation, if needed.
+        and perform feature transformation, if needed.
+
+        The actual implementation how to load is deferred to _load_series. Overwrite that for new subclasses.
 
-           The actual implementation how to load is deferred to _load_series. Overwrite that for new subclasses.
+        This function resamples data if the frequency is not equal to the expected frequency.
+        It is possible to customise this resampling (without that, we aggregate means after default resampling).
+        To customize resampling, pass in a `resampling_config` argument when you initialize a SeriesSpecs,
+        with downsampling_method and upsampling_method names (e.g. "mean" and "pad", respectively)
+        and keyword params which are to be passed into `pandas.Series.resample`.
+        For example:
 
-           This function resamples data if the frequency is not equal to the expected frequency.
-           It is possible to customise this resampling (without that, we aggregate means after default resampling).
-           To customize resampling, pass in a `resampling_config` argument when you initialize a SeriesSpecs,
-           with an aggregation method name (e.g. "mean") and kw params which are to be passed into
-           `pandas.Series.resample`. For example:
+        `resampling_config=dict(closed="left", downsampling_method="sum", upsampling_method="reverse_sum")`
```

Comment:
For clarity, I would add that
- in this example, `closed` will become an argument to `pandas.Series.resample`.
- acceptable values for `downsampling_method` and `upsampling_method` are (string names of) possible re-sample methods offered by pandas *plus* a timetomodel-version of "reverse_sum".
- "mean" and "pad" are the default values.